src/components/ppom/transactions.js

```diff
@@ -51,17 +47,17 @@ export function ppomMaliciousTransactionsAndSignatures(parentContainer) {
           </button>
           <button
             class="btn btn-primary btn-lg btn-block mb-3"
-            id="maliciousContractInteractionButton"
+            id="maliciousSetApprovalForAll"
             disabled
           >
-              Malicious Contract Interaction
+            Malicious Set Approval For All
           </button>
           <button
             class="btn btn-primary btn-lg btn-block mb-3"
-            id="maliciousSetApprovalForAll"
+            id="maliciousContractInteractionButton"
             disabled
           >
-            Malicious Set Approval For All
+            Malicious Contract Interaction
```

Comment:
re-ordered these 2 as it makes more sense like this (first all custom contract interactions, and the last one the fallback contract interaction) -- also following this order for the batch